### PR TITLE
17.2 update - address issues with initial Invoice Ninja v5.x release

### LINKED ISSUE #1778: Invoice Ninja - tweaks needed
[Lee has reported on the forums](https://www.turnkeylinux.org/forum/support/sun-20221120-1932/invoice-ninja-v5-issues) that he has had a really frustrating time trying to get the new v17.1 Invoice Ninja app working.

The issues - so far:
- It looks like lots of it's functionality depends on API callbacks, so it won't work without a domain set and a proper SSL cert. At the very least this should be documented.
- Email config should probably be set at firstboot, or at least documented.
- Provision of a `turnkey-php` or perhaps `turnkey-artisan` helper script(s).
- Default permissions seem not quite right? (See below for fix)
- PHP memory too low (needs 512M min).
- Queue not enabled - we should look at enabling that!
- PDF generation not set up for local use (still relies on 3rd party service - see below for fix).

----

##Some fixes

Fix for permissions:
```
chown -R www-data:www-data /var/www/invoiceninja
find /var/www/invoiceninja -type d -exec chmod 755 {} \;
```

Install local PDF renderer:
```
runuser -l -s /bin/bash -c "cd /var/www/invoiceninja && vendor/bin/snappdf download"
sed -i "\|^PDF_GENERATOR|s|=.*|=snappdf|" /var/www/invoiceninja/.env
runuser -l www-data -s /bin/bash -c "php /var/www/invoiceninja/artisan optimize"
```

### PR DESCRIPTION
**Pull request by JedMeister**
Closes https://github.com/turnkeylinux/tracker/issues/1778

### CODE REVIEW

File: changelog [hunk 1]
```diff
@@ -1,3 +1,24 @@
+turnkey-invoice-ninja-17.2 (1) turnkey; urgency=low
+
+  * Update build code to always install latest Invoice Ninja release.
+    Current release includes: Invoice Ninja v5.5.46
+
+  * Include turnkey-artisan helper script.
+
+  * Bump PHP memory limit to 512MB (recommended Leantime min RAM).
```
Comment (OnGle) [inline changelog]: leantime?

**Review by OnGle — APPROVED** [review]
lgtm

File: changelog (hunk 1 above)
Comment (JedMeister) [reply changelog]: Good catch, thanks man! :)

**Review by OnGle — APPROVED** [review]
Nice :+1:

**Comment by JedMeister** [comment]
Some more commits. I've been thinking that there's one more thing I'd like to implement (queue processing via supervisord), so one or 2 more commits coming...

**Review by Christie018 — APPROVED** [review]
Changes to further update invoice-ninja all look good to me. Additionally, changes have been documented in the changelog and the related tracker issues should be resolved with the code changes in this PR.

**Review by OnGle — APPROVED** [review]
lgtm

**Pull request merged**